content/articles/concept-of-first-hop-redundancy-protocol/index.md

```diff
@@ -0,0 +1,133 @@
+  - [HSRP preemption](#hsrp-preemption)
+- [Steps to configuring HSRP](#steps-to-configuring-hsrp)
+- [Configuring HSRP protocol on a Cisco router](#configuring-hsrp-protocol-on-a-cisco-router)
+- [Conclusion](#conclusion)
+- [Activity File](#activity-file)
+- [Further reading](#further-reading)
+
+### Goals
+At the end of this article, the reader is expected to learn:
+- What FHRP is
+- Various FHRP options available
+- How to configure HSRP on a Cisco router.
+  
+### Prerequisite
+To better understand this article, the reader is expected to have a basic understanding of the Cisco Command Line Interface (CLI)
+
+### Requirements
+We will use the Cisco packet tracer software for this tutorial which one can download at [Download Packet Tracer](https://www.netacad.com/portal/resources/packet-tracer)
+
+### Default gateway
+In a network environment, devices must communicate with each other within the network and outside the network. A device that communicates with over devices outside its network or subnet requires a default gateway.
+A default gateway is a network node that allows seamless communication between networks, allowing one device to communicate with another on a different network or subnet. It is the first route taken by an Internet Protocol (IP) packets to ensure that internet requests are sent to the right destination.
+The node serving as a default gateway is usually is configured with an Internet Protocol (IP) address. Each client in the network or subnet is configured with the same IP address. Because of this, if the router or the layer three switch serving as a default gateway fails, all devices connected to it will be isolated from the outside network.
+To mitigate such failures from occurring, the First Hop Redundancy Protocol (FHRP) was developed.
+
+### First Hop Redundancy Protocol (FHRP)
+FHRP is used to prevent network failure at a default gateway. This is achieved by configuring multiple routers with the same IP address and Mac address, thus presenting an illusion of a single virtual router to the hosts in a Local Area Network (LAN).
+The IP address of the virtual router is configured on all hosts in that network or subnet as their default gateway. 
+When an IP packet is sent from host devices on that network, the host uses Address Resolution Protocol (ARP) to resolve the MAC address associated. Any physically active router processes the packet sent to this MAC address within the router group. By so doing, if one router goes down, the IP packet will be processed and forwarded by the next active router.
+FHRP uses a mechanism known as First Hop Redundancy to determine which router takes an active role in forwarding traffic and when a standby router should take over this active role.
+
+### Various First Hop Redundancy Protocols
+There are various First Hop Redundancy Protocols available. Some of these protocols are proprietary, while others are standard protocols.
+
+#### Host Standby Router Protocol (HSRP)
```

Comment:
If possible add images describing each protocol.

Reply:
I see you removed content. That was meaningful and I think you should let it be.

Reply:
@lalith1403 I went through it and no content was removed. you asked me to do this https://github.com/section-engineering-education/engineering-education/pull/2489#discussion_r648489945. and that was i did. 

This PR seems to be taking too long. If you won't n be offended, i will request  that you re-assign the PR to some else, Sir.

Reply:
One last check and this should be ready for final review.